napari/_qt/qt_viewer.py

```diff
@@ -195,6 +186,22 @@ def __init__(self, viewer):
 
         self.setAcceptDrops(True)
 
+    @property
+    def console(self):
+        """QtConsole: iPython console terminal integrated into the napari GUI.
+        """
+        if self._console is None:
+            from .qt_console import QtConsole
+
+            self.console = QtConsole({'viewer': self.viewer})
```

Comment:
I just want to flag that now *might* be a good time to update this behaviour, but that can wait for a follow-up PR, since this is not making things worse.

Reply:
Sounds good

Reply:
Made a note in #787 which is still open